newIDE/app/src/ProjectManager/index.js

```diff
@@ -714,6 +533,24 @@ export default class ProjectManager extends React.Component<Props, State> {
     this._onProjectItemModified();
   };
 
+  _onEditEventsFunctionExtension = (
```

Comment:
nitpicking: you could even take the side of using a long name like  `_onEditEventsFunctionExtensionOrSeeDetails` to show the "or" part that means that something will differ according to extensions.

Reply:
Good idea